### ISSUE TITLE
Bootstrapping self-test

### ISSUE THREAD

**Issue by pblasucci** [Krzysztof-Cieslak/Aenea#1]
This more of a question/suggestion than anything else...

I see the current test suite uses Expecto. Can we (should we) get to the point where Aenea is tested using Aenea (similar to bootstrapping a compiler)?

**Comment by Krzysztof-Cieslak** [comment]
Yes, the plan is to self-test, indeed. Tests are using Expecto as this is what we currently scaffold with Waypoint template - this will probably change too.
